pokepaylib/src/main/java/jp/pokepay/pokepaylib/Response.java

```diff
@@ -7,7 +7,7 @@
 
 public class Response {
 
-    private static final SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS");
+    public static final SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS");
```

Comment:
@WeiWW why does this need to be public?

Reply:
@taniryo-kun  because it will be used [here ](https://github.com/pokepay/flutter-sdk-internal/blob/25f16ce7acfc4118af7399d8e7cce8817b6dfdd1/android/src/main/java/jp/pokepay/pokepay_sdk/PokepaySdkPlugin.java#L159)